src/index.ts

```diff
@@ -176,3 +184,35 @@ declare global {
     [BentoDocumentEvents.onComponentVisibilityChange]: OnComponentVisibilityChange;
   }
 }
+
+//
+// IntrinsicElements
+//
+
+type CommonBentoElementAttributes = {};
+
+export interface IntrinsicElements {
+  /**
+   * Used to programatically control where the inline component responsible for displaying
+   * the resource center and any onboarding guides should live, usually in a getting-started page
+   * shown to end-users after logging in.
+   *
+   * WARNING: We strongly recommend you use the auto-injection feature instead, where you determine
+   * the inline placement throught the Bento App and therefore can make changes live without
```

Comment:
Change: changes live without
To: live change without

Reply:
Will do, tks